babel/core.py

```diff
@@ -885,6 +885,41 @@ def ordinal_form(self):
         """
         return self._data.get('ordinal_form', _default_plural_rule)
 
+    @property
+    def measurement_systems(self):
+        """Localized names for various measurement systems.
+
+        >>> Locale('fr', 'FR').measurement_systems['US']
+        u'am\\xe9ricain'
+        >>> Locale('en', 'US').measurement_systems['US']
+        u'US'
+
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
+        """The text direction for the language.
+
+        >>> Locale('de', 'DE').character_order
+        'left-to-right'
+        >>> Locale('ar', 'SA').character_order
+        'right-to-left'
+        """
+        return self._data['character_order']
+
+    @property
```

Comment:
This feels pretty redundant with the method above. I understand the value of providing the CSS-specific value in this package (commonly used), but I don't think it belongs in the Locale class.

We could create helper modules that provide methods like this that transform CLDR values into ones accepted by other systems. Off hand, I can't think of any other ones that might be relevant for CSS (maybe language code?), but, for instance, converting date format strings to JS compatible ones might be useful to someone.